docs/source/Components/Mpd.rst

```diff
@@ -0,0 +1,17 @@
+MPD structure
```

Comment:
Add :orphan: and a new line here at the top to squash a warning about this not being included in any octree